src/components/GroupList.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import Group from './Group'
+import { Team } from '../mocks/SearchGroupMock'
+
+interface GroupListProps {
+  groups: Team[]
+  showMenuButton: boolean
+  onCardClick: (group: Team) => void
+  onButtonClick?: (group: Team) => void
+}
+
+const GroupList: React.FC<GroupListProps> = ({
+  groups,
+  showMenuButton,
+  onCardClick,
+  onButtonClick,
+}) => {
+  GroupList.defaultProps = {
+    onButtonClick: () => {},
+  }
```

Comment:
```suggestion
```

defaultProps는 deprecated 되었으므로 제거하는게 좋겠습니다.

- https://ko.react.dev/blog/2024/04/25/react-19-upgrade-guide

> We’re also removing defaultProps from function components in place of ES6 default parameters. Class components will continue to support defaultProps since there is no ES6 alternative.